### PR TITLE
addressed the issue #4415

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]
Follow the workflow detailed in the issue #4415 to test. Everything should work as specified in the issue.

### CODE REVIEW

**Comment by hyi** [comment]
@pkdash @sblack-usu @dtarb the issue #4415 has been addressed by this PR which involves code changes to a couple of files in the back end in addition to the change in the front end html file. I'd suggest to go ahead with the next release 1.52 without including this PR, but if any of you think it'd be good to include this small PR in the next release, I am fine with it too.

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW VULNERABILITIES | 0 |
| NEW CODE SMELLS | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |
| NEW BUGS | 2 |
| NEW SECURITY HOTSPOTS | 0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4460](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4460)

**Comment by dtarb** [comment]
@hyi if this can pass code reviews, I'd be keen to get this in the next 1.52 release.

**Comment by hyi** [comment]
@pkdash Given @dtarb 's comment above, it'd be great if you could give a quick review for this PR whenever you get a chance.

File: hs_core/models.py [hunk 1]
```diff
@@ -99,6 +99,22 @@ def clean_for_xml(s):
     return output
 
 
+def get_relation_version_res_url(resource, rel_type):
```
Comment (pkdash) [inline hs_core/models.py:102]: I think we should make this an instance method of the `BaseResource` class and check the parameter  `rel_type` value is one of the 2 allowed types.

**Review by pkdash — CHANGES_REQUESTED** [review]

**Comment by hyi** [comment]
@pkdash Good point. I have addressed your comment. See if it looks good now.

**Review by pkdash — APPROVED** [review]

**Pull request merged**